srcpkgs/Clight/template

```diff
@@ -1,15 +1,26 @@
 # Template file for 'Clight'
 pkgname=Clight
-version=4.1
+version=4.2
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config"
 makedepends="popt-devel gsl-devel libconfig-devel elogind-devel bash-completion
  dbus-devel libmodule-devel"
-depends="Clightd"
+depends="Clightd>=5.0"
 short_desc="Use your webcam as a light sensor to adjust screen backlight"
 maintainer="Dominic Monroe <monroef4@googlemail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/FedeDP/Clight"
 distfiles="https://github.com/FedeDP/Clight/archive/${version}.tar.gz"
-checksum=84156f3dfd9ab863be25d3efc8b5d6d6043ba4a0b44259c2a1cee1367b3c97aa
+checksum=fc3c4952c755bad5f4274f69f5cc17142abb51d88d7c615873e3058edc1c1137
+
+do_install() {
+	vbin build/clight
```

Comment:
Why are you overriding this?

Reply:
Ah, mistake from the very beginning. I tried to add shell completions but suddenly the binary disappeared. It wasn't until later that I added those to post_install. :)